pkg/fanal/analyzer/analyzer.go

```diff
@@ -101,7 +101,7 @@ type postAnalyzerInitialize func(options AnalyzerOptions) (PostAnalyzer, error)
 
 func RegisterPostAnalyzer(t Type, initializer postAnalyzerInitialize) {
 	if _, ok := postAnalyzers[t]; ok {
-		log.Logger.Fatalf("analyzer %s is registered twice", t)
+		log.Fatal("The analyzer is registered twice", log.String("type", string(t)))
```

Comment:
ditto

Reply:
Fixed in https://github.com/aquasecurity/trivy/pull/6466/commits/fdb559f3c57ca141bc1a211ef2b6d0950e84b135